internal/request/grpc.go

```diff
@@ -76,7 +78,20 @@ func registerRHCWorker() string {
 	defer cancel()
 
 	// Register as a handler of the "catalog" type.
-	r, err := c.Register(ctx, &pb.RegistrationRequest{Handler: "catalog", Pid: int64(os.Getpid())})
+	registrationRequest := pb.RegistrationRequest{
+		Handler: "catalog",
+		Pid:     int64(os.Getpid()),
+		Features: map[string]string{
+			"SourceRef":       viper.GetString("ANSIBLE_TOWER.uuid"),
+			"SrcName":         viper.GetString("ANSIBLE_TOWER.name"),
+			"SrcType":         "ansible-tower",
+			"ApplicationType": "/insights/platform/catalog",
+			"WorkerVersion":   build.Version,
+			"WorkerBuild":     build.Build,
+			"WorkerSHA":       build.Sha1,
```

Comment:
@bzwei Should we also add the Date when the binary was built?

Reply:
I change the Build content to built date but keep the key `WorkerBuild`